testutil/src/test/java/org/spine3/test/CommandTestShould.java

```diff
@@ -116,7 +122,32 @@ public void create_command_with_custom_Timestamp() {
         final Timestamp timestamp = Timestamps.minutesAgo(5);
         final Command command = commandTest.createCommand(commandMessage, timestamp);
 
-        assertEquals(timestamp, command.getContext().getTimestamp());
+        assertEquals(timestamp, command.getContext()
+                                       .getTimestamp());
+    }
+
+    @SuppressWarnings("ConstantConditions") // Passing `null` is the purpose of the test.
+    @Test(expected = NullPointerException.class)
+    public void do_not_accept_null_command_on_createOtherCommand() {
```

Comment:
`..._createOtherCommand` does not match the original method name, i.e. `commandTest#createAnotherCommand(...)`.